.changeset/old-flowers-cheat.md

```diff
@@ -0,0 +1,8 @@
+---
+'@contentful/f36-icon': minor
+'@contentful/f36-icons': minor
+'@contentful/f36-note': minor
+'@contentful/f36-text-link': minor
+---
+
+Add new premium variant to Note, Textlink, Icon. Add new Diamond Icon.
```

Comment:
That's what I mentioned in person yesterday, the bullet point is missing from the generated changeset.

```suggestion
- Add new premium variant to Note, Textlink, Icon. Add new Diamond Icon.
```

Reply:
ah ja! will update :)